pybamm_tea/tea.py

```diff
@@ -0,0 +1,347 @@
+#
```

Comment:
You need an `__init__.py` in this folder so you can expose the `TEA` class as part of the package